Pill trilogy: The sweet return of the successor unchained

Dimasw99 opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

The shipfreeze is no more and I fixed up the conversion.

This is an attempt to bring back the pill with its issues fixed while trying to keep its concept intact.

Ship to ship: A holopad and a wideband channel is now available
Player count: Default prisoner count is 6
Ship limit: The Sugarcube is limited to 2 ships per round

![sugarcube](https://user-images.githubusercontent.com/21098781/150725729-25a0bb0d-4e19-4afa-89b2-70c462c869a5.PNG)


<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game

The pill design philosophy unfolded shiptests potential and it was quite evident on the results. This ship should still provide the unforgiving feeling of no mistakes allowed. It should be great fun
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
add: Added Sugarcube Prison bus ship. The pill returns.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Dimasw99 committed:
sugarcube

Dimasw99 committed:
dockingport fix

Dimasw99 committed:
docking_port fix for real

Dimasw99 committed:
Merge branch 'shiptest-ss13:master' into sugarcube

Dimasw99 committed:
conversion

Zevotech commented:
Woah Its The Bus From Black Ops Zombies

Latentish commented:
Telecomms to start with seems like a bit much for a pill successor, don't you think?

Dimasw99 commented:
> 

It is a singular relay and not a 1 in all. It does not provide you with coms and is just a useless machine draining power

swifterjack commented:
> > 
> 
> It is a singular relay and not a 1 in all. It does not provide you with coms and is just a useless machine draining power

holy shit based as hell, more ships need machines that solely exist to make the power econ harder

Ms-Mee commented:
> > > 
> > 
> > It is a singular relay and not a 1 in all. It does not provide you with coms and is just a useless machine draining power
> 
> holy shit based as hell, more ships need machines that solely exist to make the power econ harder

That'd require power to actually be something that matters

Spyroshark commented:
after playing a round with a full crew, i have to say that 6 people is WAY TOO MANY FOR A SHIP OF THIS SIZE. Having a single 3x6 room for 6 different people doing different things is a horrible experience, although considering this IS a successor to the pill, maybe it should be that way.

Complementary image from a two hour Sugarcube round:
![hell](https://user-images.githubusercontent.com/35285914/154784846-dc3b381a-6166-4b46-a232-cedbc1cb7d84.PNG)

Dimasw99 commented:
> after playing a round with a full crew, i have to say that 6 people is WAY TOO MANY FOR A SHIP OF THIS SIZE. Having a single 3x6 room for 6 different people doing different things is a horrible experience, although considering this IS a successor to the pill, maybe it should be that way.

Yeah the overwhelming crew size is intial because the sugarcube is intended to be expanded but also offer plenty of respawns in a ship that is prone to deaths. The previous number used to be 3 for a pill but if people think that it is too much I may turn it down to 5.

In my play through with the sugarcube we were able to grab some EVA gear from some legions, expand the ship twice the size, get RnD going and had the true zero to hero pill experience. Likewise I may add one more pickaxe and a singular soft suit. Lastly ship to ship interaction for trade is also always possible for EVA gear.

ZephyrTFA commented on _maps/configs/sugarcube.json:
```diff
@@ -0,0 +1,11 @@
+	"map_id": "sugarcube",
+	"limit": 2,
+	"job_slots": {
+		"Prisoner": 6
```
```suggestion
		"Prisoner": 4
```
6 is still just a bit too much for a ship this small; if they want more crew they can open more slots

Dimasw99 committed:
Update _maps/configs/sugarcube.json

Co-authored-by: Zephyr <12817816+ZephyrTFA@users.noreply.github.com>

Dimasw99 committed:
Adjustments

The precharger now starts with 20% charged instead of full
Added two beers so that dwarf players don't have to sweat power game speedrun fermentation
Added a poster
Added 3 floorpills. Enjoy

Dimasw99 commented:
The precharger now starts with 20% charged instead of full
Added two beers so that dwarf players don't have to sweat power game speedrun fermentation
Added a poster
Added 3 floorpills. Enjoy
